idact/detail/jupyter/jupyter_deployment_impl.py

```diff
@@ -74,7 +74,6 @@ def cancel(self):
                                            "Cancelling Jupyter deployment."))
             stack.enter_context(cancel_on_exit(self._deployment))
             self.cancel_local()
-
```

Comment:
Why this deletion?

Reply:
It's not very similar. By and large, only environment loading